pkg/installmanager/installmanager.go

```diff
@@ -398,54 +409,59 @@ func (m *InstallManager) Run() error {
 		}
 	}
 
-	installErr := m.provisionCluster()
-	if installErr != nil {
-		m.log.WithError(installErr).Error("error running openshift-install, running deprovision to clean up")
+	if fakeCluster {
+		m.log.Warnf("skipping openshift-install create cluster due to %s annotation on ClusterDeployment",
+			constants.HiveFakeInstallKubeconfigSecretAnnotation)
+	} else {
```

Comment:
super not important:
most of the actual work is in else block now, can we switch so that actual work is in first block.. just for readability.

Reply:
Done, although I'm worried I misunderstood your previous comment.